### PR TITLE
Dltp 900 sipity figaro remediation

### PR DESCRIPTION
**Pull request by banurekha**

### CODE REVIEW

File: config/initializers/figaro.rb [hunk 1]
```diff
@@ -1,23 +1,47 @@
 Figaro.require_keys(
-  'airbrake_api_key',
```
Comment (jeremyf) [inline config/initializers/figaro.rb]: Why did you change the spacing from 2 to 4 for indentation? The Ruby convention is 2 spaces per indent.

Comment (banurekha) [inline config/initializers/figaro.rb]: Corrected.. Subl Error.. Didn't notice it.
